src/components/heroes.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
```

Comment:
- [ ] Kindly add more unit tests for your components, you only have 2 test files.